pennylane_lightning/src/benchmarks/Bench_LinearAlgebra.cpp

```diff
@@ -452,7 +452,8 @@ template <class T> static void std_scaleAndAdd_cmplx(benchmark::State &state) {
 
     std::vector<std::complex<T>> vec1;
     std::vector<std::complex<T>> vec2;
-    std::complex<T> scale{std::cos(0.4123), std::sin(0.4123)};
+    std::complex<T> scale{static_cast<T>(std::cos(0.4123)),
+                          static_cast<T>(std::sin(0.4123))};
```

Comment:
How about using `std::cos(static_cast<T>(0.4123))` instead? It will use a proper version of `std::cos` overloaded for `T`.

Reply:
Sure!